### PR TITLE
feat: add testnet components to build

### PR DESCRIPTION
**Pull request by fakirAyoub**
Add step for generating wasm and ci and add them to the artifacts in CI

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -94,6 +94,31 @@ jobs:
...
+      - name: Generate wasm binary for testnet
+        run: ./target/release/hydradx export-genesis-wasm --chain hydradx-testnet-chainspec.json > para-wasm
+
+      - name: Generate genesis binary for testnet
+        run: ./target/release/hydradx export-genesis-state --chain hydradx-testnet-chain > para-genesis
+
+      - name: Upload wasm binary
+        uses: actions/upload-artifact@v2
+        with:
+          name: para-wasm
```
Comment (lumir-mrkva) [inline .github/workflows/tests.yml:114]:
```suggestion
          name: devnet-para-wasm
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -94,6 +94,31 @@ jobs:
...
+          path: ./para-wasm
+
+      - name: Upload genesis binary
+        uses: actions/upload-artifact@v2
+        with:
+          name: para-genesis
```
Comment (lumir-mrkva) [inline .github/workflows/tests.yml:120]:
```suggestion
          name: devnet-para-genesis
```

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]
build has to succeed

**Comment by fakirAyoub** [comment]
> build has to succeed

@lumir-mrkva the build has been fixed. Thank you. This PR can now be merged.

**Pull request merged**